sdks/java/io/snowflake/src/main/java/org/apache/beam/sdk/io/snowflake/Location.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.io.snowflake;
+
+import java.io.Serializable;
+
+/**
+ * Used as one of the arguments for {@link org.apache.beam.sdk.io.snowflake.SnowflakeIO} write and
+ * read method.
```

Comment:
Would be nice to also briefly mention what purpose it serves, e.g. providing the location that will be read from/written to.

Reply:
I changed to 
```
/**
 * Used as one of the arguments for {@link org.apache.beam.sdk.io.snowflake.SnowflakeIO} write and read operations.
 * It keeps information about storage integration and staging bucket name.
 * Storage integration is the name of a Snowflake storage integration object created according to Snowflake documentation for the GCS bucket.
 * Staging bucket name is name of the Google Cloud Storage bucket which in the case of writing operation will be used to save CSV files
 * which will end up in Snowflake under “staging_bucket_name/data” path and in the case of reading operation will be used as a temporary location
 * for storing CSV files named `sf_copy_csv_DATE_TIME_RANDOMSUFFIX` which  will be removed automatically once Read operation finishes.
 *
 */
```